Add Rehau integration documentation (#13172)
home-assistant/home-assistant.github.io: source/_integrations/rehau_nea_smart_manager.markdown

---
title: "Rehau Nea Smart Manager"

> description: ""

Reviewer: Description missing.

Author: Fixed.

> ha_release: "0.0.1"

Reviewer 2: ```suggestion
ha_release: "0.110"
```

Author: Fixed.

ha_category: Climate
ha_iot_class: "Local Polling"

> ha_quality_scale: internal

Reviewer: This is not an internal component of Home Assistant.

Author: Fixed.

> ha_config_flow: true

Reviewer: The description below does not suggest it is using a configuration flow but YAML configuration?

ha_codeowners:
  - '@Jeoffreybauvin'
ha_domain: rehau
---

Integrates [Rehau Nea Smart Manager](https://www.rehau.com/en-en/nea-smart/francais) Thermostat controller into Home Assistant.

> This integration generates a climate entity for each heatarea in your Nea Smart Manager.

Reviewer: ```suggestion
This integration generates a climate entity for each heating area in your Nea Smart Manager.
```

Author: Fixed.

To enable this platform, add the following lines to your `configuration.yaml` file:

```yaml
# Example configuration.yaml entry
climate:

>   - platform: rehau

Reviewer: We don't allow new platform based integrations. See ADR-0007:

<https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md>

Furthermore, this integration communicates with a device or service, so ADR-0010 also applies:

<https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md>

>     host: '192.168.1.2'

Reviewer 2: ```suggestion
    host: IP_ADDRESS
```

Author: Fixed.

```

<div class='note warning'>
For security reasons, this integration is not compatible (yet) with the cloud version of Rehau Nea Smart Manager. Please use the local IP of your Nea Smart Manager.
</div>

{% configuration %}
host:
  description: Your Nea Smart Manager local IP (or DNS name).
  required: true
  type: string
{% endconfiguration %}

## Preset modes

| Preset mode | Description |
| ---------------------- | -------- |
| `auto` | Auto mode : use Nea Smart Manager scheduling
| `eco` | Eco mode
| `comfort` | Comfort mode

> ## Component services

Reviewer: This section only contains services provided by the climate component. Hence, they should not be documented here as they are documented by the climate integration itself.

Author: Removed, thanks.
